README.md

```diff
@@ -101,7 +101,7 @@ No one likes to argue about code formatting in pull requests, as project we take
 Build with the `dist` profile as shown above, then execute this:
 
 ```
-$ java -jar kroxylicious/target/kroxylicious-*-SNAPSHOT.jar -cp {path-to-your-class-path} --config {path-to-kroxylicious-config}
+$ java -cp {path-to-your-class-path} -jar kroxylicious/target/kroxylicious-*-SNAPSHOT.jar --config {path-to-kroxylicious-config}
```

Comment:
I think we should drop the `-cp` bit as you've seen evidence it isn't used right? Like if you have an slf4j implementation in your `-cp` it isn't picked up. It sounds like `-cp` and `-jar` are incompatible (very quietly -.-)

[https://docs.oracle.com/javase/8/docs/technotes/tools/findingclasses.html](https://docs.oracle.com/javase/8/docs/technotes/tools/findingclasses.html#:~:text=loaded%20is%20undefined.-,How%20the%20Java%20Launcher%20Finds%20User%20Classes,-User%20classes%20are)

So we might need another block like `### Run natively with additional libs` and a different command just using `-cp`

Reply:
> the -jar option, which overrides all other values. If this option is used, all user classes must come from the specified archive.

Hmm that does change things a bit. So our only option to run with extra libs is to build a new JAR?

Happy to remove `-cp`, but would be good to work out a viable alternative.

Reply:
You should be able to run with the same jar without using `-jar` by putting it on the classpath instead, like: ```java -cp ${path-to-your-class-path}:kroxylicious/target/kroxylicious-0.2.0-SNAPSHOT.jar io.kroxylicious.proxy.Kroxylicious --config kroxylicious/example-proxy-config.yml```

Reply:
Ah true. I've added a second run command now to cover that.

Reply:
Annoying that using the `Main-Class` attribute in the jar manifest breaks stuff.

Reply:
Yeah it's interesting, `java -cp kroxylicious/target/kroxylicious-*-SNAPSHOT.jar` must understand how to add the other jars to the classpath from the manifest, but in `-jar` mode it exclusively considers the manifest.